canvas_account_admin_tools/settings/local.py

```diff
@@ -14,8 +23,9 @@
 CANVAS_EMAIL_NOTIFICATION['support_email_address'] = 'tltqaemails@g.harvard.edu'
 
 ALLOWED_HOSTS = ['*']
-INSTALLED_APPS += ['debug_toolbar', 'django_extensions']
-MIDDLEWARE += ['debug_toolbar.middleware.DebugToolbarMiddleware']
+INSTALLED_APPS += ['django_extensions']
+# INSTALLED_APPS += ['debug_toolbar']
+# MIDDLEWARE = ['debug_toolbar.middleware.DebugToolbarMiddleware'] + MIDDLEWARE
```

Comment:
Are we moving forward with removing debug_toolbar? If so maybe remove it from local requirements

Reply:
I'll leave it in the requirements file but add a comment here about how to enable it.